language/move-lang/src/expansion/translate.rs

```diff
@@ -632,24 +632,33 @@ fn spec_member(
         PM::Include {
             name: pn,
             type_arguments: ptys_opt,
-            renamings,
+            arguments: parguments,
         } => {
             let name = module_access(context, pn)?;
             let type_arguments = optional_types(context, ptys_opt);
+            let arguments = parguments
+                .into_iter()
+                .map(|(n, e)| (n, *exp(context, e)))
+                .collect();
             EM::Include {
                 name,
                 type_arguments,
-                renamings,
+                arguments,
             }
         }
         PM::Apply {
             name: pn,
             type_arguments: ptys_opt,
+            arguments: parguments,
             patterns: ppatterns,
             exclusion_patterns: pe_patterns,
         } => {
             let name = module_access(context, pn)?;
             let type_arguments = optional_types(context, ptys_opt);
+            let arguments = parguments
+                .into_iter()
+                .map(|(n, e)| (n, *exp(context, e)))
```

Comment:
There is a `exp_` helper so you don't have to box, then unbox. So `exp_(context, e)`

Reply:
Done